src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -232,7 +232,7 @@
 		<Setter Property="BorderBrush"
 				Value="{ThemeResource SystemControlForegroundChromeDisabledLowBrush}" />
 		<Setter Property="SelectionHighlightColor"
-				Value="{ThemeResource SystemControlHighlightAccentBrush}" />
+				Value="{ThemeResource TextControlSelectionHighlightColor}" />
```

Comment:
The doc says it's `TextSelectionHighlightThemeColor`
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textbox.selectionhighlightcolor?view=winrt-22000#property-value

Reply:
Seems the actual default is different in Generic.xaml in Windows Kits 🤷‍♂️🤷‍♂️